spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'simplecov'
+require 'factory_girl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.